src/controllers/userController.js

```diff
@@ -19,5 +19,32 @@ const User = {
     const data = { ...userData, token };
     return sendResult(res, 201, 'Account created successfully', data);
   },
+  async login(req, res) {
+    const { email, password } = req.body;
+    if (!email || !password || !(email && password)) {
+      return sendResult(res, 401, 'Both email and password are required');
+    }
+    const userInfo = await db.Users.findOne({
+      where: {
+        email,
+      },
+    });
+    if (!userInfo) {
+      return sendResult(res, 401, 'Univerifed account or the email and/or password is invalid');
+    }
+
+    const comfirmPass = helpers.comparePassword(password, userInfo.password);
+    if (comfirmPass) {
+      const token = helpers.createToken(userInfo.id, userInfo.email, false);
+      const {
+        id, username,
+      } = userInfo;
+      const data = {
+        id, username, email, token,
+      };
+      return sendResult(res, 201, 'User logged successfully', data);
+    }
+    return sendResult(res, 401, 'Univerifed account or the email and/or password is invalid');
```

Comment:
nice try. again try to make decent functionality to check if a user is verified like
`if (!userInfo.isverified)` and send a relevant message, that advice the user to verify their account.

Reply:
I did it